### PR TITLE
chore: remove legacy comment

### LINKED ISSUE #2532: Refactor: replace inline new StringMap invocations with calls to classes
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

I had to solve it via inlining the solution because adding new methods leads to a breaking change for python

### References

- #0000

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
This was solved in #2375 already

Closes #2532

### CODE REVIEW

File: packages/cdktf/lib/terraform-resource.ts [hunk 1]
```diff
@@ -22,6 +22,12 @@ import {
   LocalExecProvisioner,
   RemoteExecProvisioner,
 } from "./terraform-provisioner";
+import {
+  AnyMap,
+  BooleanMap,
+  NumberMap,
+  StringMap,
+} from "./complex-computed-list";
```
Comment (ansgarm) [inline packages/cdktf/lib/terraform-resource.ts:30]: Why is this added?

Comment (DanielMSchmidt) [reply packages/cdktf/lib/terraform-resource.ts:30]: Didn’t cmd + z enough 😅

**Review by ansgarm — APPROVED** [review]
